src/countlyInterceptor.js

```diff
@@ -26,7 +28,17 @@ const handleError = (/** @type {unknown} */ error) => {
       ? error
       : new CLIError(`Error: ${String(error ?? "no error?")}`);
 
-  console.error(err);
+  if ("stack" in err) {
```

Comment:
I suggest to extract this block of code into a helper method. Would be easier to read this way.